internal/tiltfile/docker_compose.go

```diff
@@ -60,6 +60,38 @@ type dcConfig struct {
 type dcServiceConfig struct {
 	RawYAML []byte        // We store this to diff against when docker-compose.yml is edited to see if the manifest has changed
 	Build   dcBuildConfig `yaml:"build"`
+	Volumes Volumes
+}
+
+type Volumes struct {
+	Volumes []Volume
+}
+
+type Volume struct {
+	Source string
+}
+
+func (v *Volumes) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
not excited about implementing our own marshalling logic here. i think there should be a library for this? but i haven't spent much time looking into how to leverage this

https://github.com/docker/cli/tree/master/cli/compose

Reply:
going to tackle this in a separate PR